src/main/java/raisetech/StudentManagement/Student_course.java

```diff
@@ -0,0 +1,16 @@
+package raisetech.StudentManagement;
+
+import java.time.LocalDateTime;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Student_course {
```

Comment:
クラスの命名がおかしいですね 👀 
ファイル名も合わせて修正しましょう！
```suggestion
public class StudentCourse {
```